app/minigames/urls/conversations/1-intro.oulipo

```diff
@@ -13,63 +15,63 @@ Arca:
 
 # passage
 set asked-passage
-Hobo [irritated]: I'm afraid I can't do that. This is my town. I'm the mayor. Completely legit. Understand?
+Hobo [irritated track:02f-dont-pass]: I'm afraid I can't do that. This is my town. I'm the mayor. Completely legit. Understand?
 go menu-kittens
 
 # kittens
 set asked-kittens
-Hobo [worried]: Strange ya mentioned that. A funny lookin' chap with a tail came an' asked where he could find kittens, just t'other day. Didn't like the look of his jumper, so I told 'im to bob off.
+Hobo [worried track:03d-floyd]: Strange ya mentioned that. A funny lookin' chap with a tail came an' asked where he could find kittens, just t'other day. Didn't like the look of his jumper, so I told 'im to bob off.
 Arca:
 	- Oh... did he say why he was looking for them?
 		continue
 	- Did you get his name?
 		continue
 	- Do you know where he was going?
 		continue
-		
+
 Hobo [irritated]: No
```

Comment:
Yeah the 04f-didnt-say.mp3 actually says: "No, he didn't say" shall I add in the text?